### ISSUE TITLE
Error after sending feed and duplicite feedbacks

### ISSUE THREAD

**Issue by radimjanda754** [satismeter/satismeter-android#3]
Hello every time after I try to send feedback I get this error:
```
E/SatisMeter: org.json.JSONException: End of input at character 0 of 
        at org.json.JSONTokener.syntaxError(JSONTokener.java:449)
        at org.json.JSONTokener.nextValue(JSONTokener.java:97)
        at org.json.JSONObject.<init>(JSONObject.java:159)
        at org.json.JSONObject.<init>(JSONObject.java:176)
        at com.satismeter.reqests.base.HttpRequestExecutor.onPostExecute(HttpRequestExecutor.java:84)
        at com.satismeter.reqests.base.HttpRequestExecutor.access$100(HttpRequestExecutor.java:16)
        at com.satismeter.reqests.base.HttpRequestExecutor$1.onPostExecute(HttpRequestExecutor.java:36)
        at com.satismeter.reqests.base.HttpRequestExecutor$1.onPostExecute(HttpRequestExecutor.java:30)
        at android.os.AsyncTask.finish(AsyncTask.java:695)
        at android.os.AsyncTask.access$600(AsyncTask.java:180)
        at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```
Also every time I click on cross after sending feedback, I see the feedback duplicitly 2 times in the web app.

Im using latest version `com.satismeter:SatisMeter:1.1.3`

**Comment by jsedlacek** [comment]
Thanks for the bug report. @radimjanda754 Can you please send me your write key, so we can check what went wrong?

**Comment by radimjanda754** [comment]
@jsedlacek Thank you, I have sent our key to support@satismeter.com if thats ok

**Comment by MartinNuc** [comment]
Any news here?

**Comment by jsedlacek** [comment]
@MartinNuc we have a fix, but not released yet. Let me publish a new version.

**Comment by jsedlacek** [comment]
@MartinNuc published under version 1.1.4. Please let us know if it fixed the issue with `JSONException` being logged.

**Comment by radimjanda754** [comment]
Thank you, I do no longer see the exception, but I still see the problem with duplicite answers in web app as I also mentioned, I thought it maybe is related to the exception.

How I can reproduce this:
After sending answer there is thank you layout at bottom, when I just wait for it to dissapear, then everything seems ok. But when I shut down this layout earlier (like by clicking on cross button or clicking into app) then the layout dissapears immediately but the answer for some reason gets duplicated (2 times), I can see it in web app.

**Comment by jsedlacek** [comment]
Hi @radimjanda754, thanks for sending over the instructions to reproduce this behavior.

We'll let you know once we have a fix for that.

Meanwhile, we have a workaround - we can set up a check on your SatisMeter project, that will ignore these duplicate responses. Just let us know via our support live chat.

**Comment by jsedlacek** [comment]
@radimjanda754 fixed in version 1.2.0.
